stubs/opentracing/opentracing/mocktracer/tracer.pyi

```diff
@@ -1,10 +1,26 @@
+from typing import Any
+
 from ..scope_manager import ScopeManager
 from ..span import Span
-from ..tracer import Tracer
+from ..tracer import Reference, Tracer
+from .context import SpanContext
 from .propagator import Propagator
+from .span import MockSpan
 
 class MockTracer(Tracer):
     def __init__(self, scope_manager: ScopeManager | None = ...) -> None: ...
+    @property
+    def active_span(self) -> MockSpan | None: ...
     def register_propagator(self, format: str, propagator: Propagator) -> None: ...
-    def finished_spans(self) -> list[Span]: ...
+    def finished_spans(self) -> list[MockSpan]: ...
     def reset(self) -> None: ...
+    def start_span(
+        self,
+        operation_name: str | None = ...,
+        child_of: Span | SpanContext | None = ...,
```

Comment:
This `SpanContext` is not the same `SpanContext` as the one in `Tracer`. Is this intentional? If not, change `from .context import SpanContext` to `from ..context import SpanContext`.

Yes, having two classes with the same name is confusing. And mypy's error messages that have only the class name in them don't exactly help :)

Reply:
So all mock should now use `from .context import SpanContex`